elliottlib/cli/find_cve_flaws_cli.py

```diff
@@ -0,0 +1,115 @@
+import bugzilla
+import click
+from errata_tool import Erratum
+from elliottlib import Runtime, bzutil
+from elliottlib.cli.common import cli, use_default_advisory_option, find_default_advisory
+from elliottlib.find_cve_flaws import get_attached_tracker_bugs, \
+    get_corresponding_flaw_bugs, is_first_fix, is_security_advisory, \
+    get_highest_security_impact, is_advisory_impact_smaller_than
+
+
+pass_runtime = click.make_pass_decorator(Runtime)
+
+
+@cli.command('find-cve-flaws',
```

Comment:
If this is designed to always attach flaw bugs (except the dry-run mode), should it be renamed to something like `attach-cve-flaws`?

Reply:
agree, renamed the command and corresponding files/functions: https://github.com/openshift/elliott/pull/158/commits/9afe40c8e36b4afe3b4612306e5438fe9dd4daf9